app/controllers/registrations_controller.rb

```diff
@@ -13,6 +13,7 @@ def create
     @user.family = family
 
     if @user.save
+      Transaction::Category.create_default_categories(@user.family)
```

Comment:
I have considered moving this into an `after_create` callback in the Family model but thought it felt strange having the Family model create other model's instances as a side effect, so I made it explicit like this.